docs/src/apps/rent.md

```diff
@@ -1 +1,45 @@
-# Accounts and Rent
+# Storage Rent for Accounts
+
+Keeping accounts alive on Solana incurs a storage cost called _rent_ because the cluster must actively maintain the data to process any future transactions on it. This is different from Bitcoin and Ethereum, where storing accounts doesn't incur any costs.
+
+The rent is debited from an account's balance by the runtime upon the first access in the current epoch by transactions or once per an epoch if there are no transactions. The rent is paid upfront for the next epoch. If an account can't pay rent for the next epoch, it's purged immediately at the start of a new epoch. The fee is currently a fixed rate, measured in bytes-times-epochs. The fee may change in the future.
+
+Accounts can be exempt from paying rent if they maintain a minimum balance. This rent-exemption is described below.
+
+## Calculation of rent
+
+Note: The rent rate can change in the future.
+
+As of writing, the fixed rent fee is 19.055441478439427 lamports per byte-epoch on the testnet and mainnet-beta clusters. An [epoch](../terminology.md#epoch) is roughly 2 days.
+
+The rent calculation includes account metadata (address, owner, lamports, etc) in the size of an account. Therefore the smallest an account can be for rent calculations is 128 bytes.
+
+For example, an account is created with the initial transfer of 10,000 lamports and no additional data. Rent is immediately debited from it on creation, resulting in a balance of 7,561 lamports:
+
+
+```
+Rent: 2,439 = 19.055441478439427 (rent rate) * 128 bytes (minimum account size) * 1 (epoch)
+Account Balance: 7,561 = 10,000 (transfered lamports) - 2,439 (this account's rent fee for an epoch)
+```
+
+The account balance will be reduced to 5,122 lamports at the next epoch even if there is no activity:
+
+```
+Account Balance: 5,122 = 7,561 (current balance) - 2,439 (this account's rent fee for an epoch)
+```
+
+Accordingly, a minimum-size account will be immediately removed after creation if the transferred lamports are less than or equal to 2,439.
+
+## Rent exemption
+
+Alternatively, an account can be made entirely exempt from rent collection by depositing at least 2 years-worth of rent. This is checked every time an account's balance is reduced and rent is immediately debited once the balance goes below the minimum amount.
```

Comment:
@aeyakovenko @t-nelson A bit dated discord discussion but this is still true... As the pr's text implicitly says, if the minimum amount threshold changes due to underlying rent fee change (heh, this is still not a thing), the once rent-exempt account may become not exempt suddenly.

Should I create an issue for this? I couldn't found one: https://discord.com/channels/428295358100013066/489504501749907468/671877112638210069

```
[8:56 AM]  trent: If its balance falls below rent-exempt min balance it will be charged again.  Today that's managed per-program
[8:56 AM]  trent: @anatoly :point_up:
[9:32 AM]  anatoly: @trent is rent-exempt min updated after creation?
[9:36 AM]  trent: @anatoly I don't see where lamports_per_byte_year is ever changed today, so no
[9:37 AM]  anatoly: @trent is that a global?
[9:37 AM]  anatoly: When do we check that it fell below?
[9:38 AM]  anatoly: as a developer, do I have any way to create a state that lasts forever regardless of rent cost changing in the future?
[9:38 AM]  trent: That's a member of the Rent struct, it's assigned a global by default though https://github.com/solana-labs/solana/blob/master/sdk/src/rent.rs#L35
[9:39 AM]  trent: I don't believe rent-exemption is hardened against the price of rent moving today, no?
[9:40 AM]  anatoly: That’s busted.
[9:41 AM]  anatoly: Something got lost in translation :slight_smile:
```

Reply:
@ryoqun It's mentioned here, https://github.com/solana-labs/solana/issues/8145.  Might be worth changing the title to something more likely to come up in search

Reply:
Thanks for the tip!